helper.rb

```diff
@@ -0,0 +1,14 @@
+require "talk_like_a_pirate"
+
+
+def piratize(normal_sentence)
+  words_to_remove = ["gold", "treasure", "coin", "coins"]
+  arry_words = normal_sentence.split - words_to_remove
+  priate_talk = arry_words.join(" ")
+  return TalkLikeAPirate.translate(priate_talk)
+end
+
+def numberConverter(number)
+  top_num = number * 8
```

Comment:
- Any floats should be converted to the nearest rational where 8 is the denominator, eg 2.5 becomes 20/8 and 0.3 becomes 2/8

You would need to check if the argument is a Float before converting as stipulated in the bullet above.